templates/components/form/fields_macros.html.twig

```diff
@@ -345,6 +346,11 @@
    {{ _self.field(name, field, label, options) }}
 {% endmacro %}
 
+{% macro emailField(name, value, label = '', options = {}) %}
+   {% set options = {'type': 'email'}|merge(options) %}
```

Comment:
maybe sthing like:
```suggestion
   {% set options = {'type': 'email', 'multiple': options.multiple ?? ''}|merge(options) %}
```

Reply:
Unless I'm missing something, the other multiple ref can't be removed: it is what actually write the 'multiple' option into the html input properties.

Reply:
I really do not know... @orthagh?

Reply:
after checking, the initial proposal is indeed possibly the best

Reply:
OK, it's good for me then